frontend/src/app/lib/addon.js

```diff
@@ -46,8 +46,9 @@ export function installAddon(store, eventCategory, experimentTitle) {
   };
 
   if (useMozAddonManager) {
-    sendToGA('event', gaEvent);
     mozAddonManagerInstall(downloadUrl).then(() => {
+      gaEvent.eventValue = RESTART_NEEDED ? 1 : 0;
```

Comment:
Is this just casting a Boolean to an int? And if so, couldn't you just do `gaEvent.eventValue = Number(RESTART_NEEDED);`?

```sh
$ node -e console.log(Number(true));  # 1
$ node -e console.log(Number(false)); # 0
```